lms/resources/_js_config.py

```diff
@@ -317,13 +342,17 @@ def _blackboard_files_available(self):
         return self._request.feature("blackboard_files")
 
     def _canvas_files_available(self):
-        """Return True if the Canvas Files API is available to this request."""
+        """
+        Return True if the Canvas Files API is available to this request.
+
+        :raise ConsumerKeyError: if request.lti_user.consumer_key isn't in the DB
+        """
```

Comment:
This could raise `ConsumerKeyError` before and it still can now, just added it to the docstring